Patches/RimSec-Security/AlienBase.xml

```diff
@@ -0,0 +1,266 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationSequence">
```

Comment:
What's up with the extra PatchOpSequence here?

Reply:
That's an oopsie. Will fix.